src/pages/en/reference/directives-reference.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Directives Reference
+---
+
+Directives are special properties that can be passed to HTML elements, Astro components, And UI Framework components. Directives can only be used inside of Astro components.
```

Comment:
```suggestion
Directives are special properties that can be passed to HTML elements, Astro components, And UI Framework components. 

Directives are Astro-specific attributes, used in your `.astro` component to "direct" Astro as it renders your HTML template.
```

(Is that accurate to say? I like linking directives with attributes because I think that's a more commonly-known word to less-experienced developers.)